custom_components/tuya_local/devices/reiga_52_fan_light.yaml

```diff
@@ -0,0 +1,77 @@
+name: Reiga Ceiling Fan
+products:
+  - id: g0ewlb1vmwqljzji
+    name: Reiga 52 Ceiling Fan with Light
+primary_entity:
+  entity: fan
+  translation_key: fan_with_presets
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+    - id: 2
+      type: string
+      name: preset_mode
+      mapping:
+        - dps_val: normal
+          value: normal
+        - dps_val: nature
+          value: nature
+        - dps_val: sleep
+          value: sleep
+    - id: 3
+      type: integer
+      name: speed
+      range:
+        min: 1
+        max: 6
+      mapping:
+        - scale: 0.06
+    - id: 8
+      type: string
+      name: direction
+secondary_entities:
+  - entity: select
+    name: Timer
+    icon: "mdi:timer"
+    category: config
+    dps:
+      - id: 22
+        type: string
+        name: option
+        mapping:
+          - dps_val: "off"
+            value: "Off"
+          - dps_val: "1hour"
+            value: "1 hour"
+          - dps_val: "2hour"
+            value: "2 hours"
+          - dps_val: "4hour"
+            value: "4 hours"
+          - dps_val: "8hour"
+            value: "8 hours"
+  - entity: light
+    name: Light
+    icon: "mdi:lightbulb"
+    category: config
+    dps:
+      - id: 15
+        type: boolean
+        name: switch
+      - id: 16
+        type: integer
+        name: brightness
+
+        mapping:
+          - scale: 0.392156862745098
+      - id: 17
+        type: integer
+        name: color_temp
+        mapping:
+          - step: 1
```

Comment:
Remove the - step: 1 line, as it will interfere with the following line as  it is a separate entry in the list and neither specifies a dps_val, so both are competing to be the default mapping, but anyway 1 is the default step so it is not needed.

Reply:
Thanks. removed